dashboard/modules/reporter/reporter_agent.py

```diff
@@ -30,6 +30,7 @@
 
 # Are we in a K8s pod?
 IN_KUBERNETES_POD = "KUBERNETES_SERVICE_HOST" in os.environ
+ENABLE_GPU_USAGE_CHECK = True
```

Comment:
Maybe we shouldn't use all upper case variable here? (I think this means the constant variable in our convention).